emsdk.py

```diff
@@ -1968,11 +1968,7 @@ def install_tool(self):
     elif url.endswith(ARCHIVE_SUFFIXES):
       success = download_and_unzip(url, self.installation_path(), filename_prefix=getattr(self, 'zipfile_prefix', ''))
     else:
-      dst_file = download_file(urljoin(emsdk_packages_url, self.download_url()), self.installation_path())
-      if dst_file:
-        success = True
-      else:
-        success = False
+      assert False
```

Comment:
we could put an error message here, even if it's just developer-facing.

Reply:
Oops, sorry, will land followup.